### PR TITLE
Bug 1372338 edited step for making ramp node highly-available

### PR DESCRIPTION
**Pull request by bfallonf**
As per: https://bugzilla.redhat.com/show_bug.cgi?id=1372338

@jkaurredhat As discussed. Do you think this fulfills the BZ?

### CODE REVIEW

**Comment by bfallonf** [comment]
Thumbs up given over IRC.

@tpoitras @ahardin-rh @adellape Peer review please?

**Comment by bfallonf** [comment]
@tpoitras @adellape @ahardin-rh bump

**Comment by bfallonf** [comment]
[rev_history]
|xref:../routing_from_edge_lb.adoc#install-config-routing-from-edge-lb[Routing from Edge Load Balancers]
|Clarified the final step of the "On the Ramp Node" section that it is optional.
%

File: install_config/routing_from_edge_lb.adoc [hunk 1]
```diff
@@ -198,8 +198,10 @@ gateway):
 ----
 ====
 
-. Mark the ramp node as an unschedulable node so that no pods end up on the ramp
-node itself:
+. Optionally, if you do not plan on configuring the ramp node to be
+highly-available, mark the ramp node as unschedulable. Skip this step if you do
```
Comment (ahardin-rh) [inline install_config/routing_from_edge_lb.adoc]: You can drop the hyphens in highly-available (both instances) per the [IBM Style Guide](http://www.ibm.com/developerworks/library/styleguidelines/).

**Pull request merged**